server/geoip_service.py

```diff
@@ -39,12 +39,12 @@ def __init__(self):
         self.db = None
         self.db_update_time = None
 
-        self.check_geoip_db_file_updated()
-
     def refresh_file_path(self):
         self.file_path = config.GEO_IP_DATABASE_PATH
 
     async def initialize(self) -> None:
+        self.check_geoip_db_file_updated()
```

Comment:
Because it reads from a file and then it logs to the console. So it's a bit weird when it's logging to the console before initialization actually starts.